installer/bundle/bundle_skel.sh

```diff
@@ -881,7 +887,8 @@ case "$installMode" in
             # Install SCX (and OMI)
             [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
             [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
-            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            [ -n "${enableOMFlag}" ] && ENABLE_OM="--enable-opsmgr" || ENABLE_OM=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG $ENABLE_OM
```

Comment:
That said, we're going to have to have a stable supported version of SCX published in [scxcore-kits][] for OMS to pick this up.

SCX v1.6.3-212 is pushed to [scxcore-kits][], and that supports it. So we're good!

[scxcore-kits]: https://github.com/Microsoft/scxcore-kits